judgels-frontends/raphael/src/routes/uriel/contests/single/SingleContestRoutes.tsx

```diff
@@ -98,6 +99,14 @@ const SingleContestRoutes = (props: SingleContestRoutesProps) => {
       component: ContestContestantsPage,
       disabled: !visibleTabs || visibleTabs.indexOf(ContestTab.Contestants) === -1,
     },
+    {
+      id: 'supervisors',
+      titleIcon: 'take-action',
+      title: 'Supervisors',
+      routeComponent: Route,
+      component: ContestSupervisorsPage,
+      disabled: !visibleTabs || visibleTabs.indexOf(ContestTab.Supervisors) === -1,
```

Comment:
I think you need to also modify the backend to return this tab. (Only show the tab if `ContestRoleChecker::canManage()`)